notebooks/noj_book/echarts.clj

```diff
@@ -43,10 +43,43 @@
   :xAxis {:data ["Shirts", "Cardigans", "Chiffons", "Pants", "Heels", "Socks"]}
   :yAxis {}
   :series [{:name "sales"
-            :type "bar"
+            :type :bar
             :data [5, 20, 36, 10, 10, 20]}]}
  {:style {:height "200px"}})
 
+;; Note that these styles are on Kindly's side,
+;; so they can't instruct on what styles would be applied to charts themselves.
+
+;; ### ECharts' Option Object
+;; It looks like there isn't docs yet about the option object passed to
+;; [setOption](https://echarts.apache.org/en/api.html#echartsInstance.setOption)
+;; at Echarts' website,
+;; and yet it's important to know what options are available out there.
+
+;; So, here we've collected some info about keys and values of this vital object:
+;; - :xAxis, whose value is an object with these keys:
```

Comment:
In comments, it is recommended to wrap keywords and symbols with [backtick](https://en.wikipedia.org/wiki/Backtick)s. 
This way, they render better in the resulting page.
`:xAxis` rather than :xAxis.

Reply:
ok, no problem.